frontend/views/containers/modals/JoinGroup.vue

```diff
@@ -0,0 +1,216 @@
+<template lang='pug'>
+modal-base-template
+  i18n(tag='h1') How to join a group
+
+  .wrapper
+    .slider
+      .slide(v-observer:0='updateIndicator' :id='config[0]')
+        .slide-img
+          svg-invitation
+
+        i18n(tag='h4') 1. Get an invitation
+        i18n(tag='p')
+          | For now, the only way to join a group is to get an invitation. Don’t know anyone using Group Income?
+        i18n(tag='button' class='link') Create your own group
+
+      .slide(v-observer:1='updateIndicator' :id='config[1]')
+        .slide-img
+          svg-proposal
+
+        i18n(tag='h4') 2. Wait for the group vote
+        i18n(tag='p') On Group Income, every major decision goes through a voting process. This includes adding new members.
+
+      .slide(v-observer:2='updateIndicator' :id='config[2]')
+        .slide-img
+          svg-access
+
+        i18n(tag='h4') 3. Use your unique access link
+        i18n(tag='p') Once the group agrees that you should join them, a unique access link will be generated, giving you instant access to the group
+
+    .dots
+      a.dot(v-for='(link, i) in config' :class='{ active: indicator == i }' :href='"#"+link')
+</template>
+
+<script>
+import ModalBaseTemplate from '@components/Modal/ModalBaseTemplate.vue'
+import { REPLACE_MODAL } from '@utils/events.js'
+import sbp from '~/shared/sbp.js'
+import SvgAccess from '@svgs/access.svg'
+import SvgInvitation from '@svgs/invitation.svg'
+import SvgProposal from '@svgs/proposal.svg'
+
+export default {
+  name: 'JoinGroupModal',
+  components: {
+    ModalBaseTemplate,
+    SvgAccess,
+    SvgInvitation,
+    SvgProposal
+  },
+  data () {
+    return {
+      indicator: 0,
+      config: [
+        'get-an-invitation',
+        'wait-for-you-group-vote',
+        'use-your-unique-access-link'
+      ]
+    }
+  },
+  // TODO move this into slider component if we have more than one
+  directives: {
+    observer: {
+      inserted: (el, { value, arg }) => {
+        try {
+          const io = new window.IntersectionObserver(
+            (elements) => {
+              if (elements[0].intersectionRatio >= 0.5) { // Wait for the middle of the screen
+                value(arg)
+              }
+            },
+            { threshold: [0.5] }
+          )
+
+          io.observe(el)
+
+          el.stop = () => {
+            io.disconnect()
+          }
+        } catch (error) {
+          console.error(error.message, '\n polyfill: https://github.com/w3c/IntersectionObserver')
+        }
+      }
+    },
+    unbind (el) {
+      el.stop()
+    }
+  },
+  methods: {
+    showLoginModal () {
+      sbp('okTurtles.events/emit', REPLACE_MODAL, 'CreateGroup')
+    },
+    updateIndicator (i) {
+      this.indicator = i
+      history.pushState(null, null, `#${this.config[i]}`)
```

Comment:
I think the link doesn't need to change based on the carousel position... @mmbotelho what do you think?

Reply:
well I did it because I think so

Reply:
Maybe I need to spend time explaining.. When you click on a dot it change the url id. So to stay consistent, it's better if the url follow on scroll too.
Now I'm not sure why you would like to comment a feature which is a nice to have and already implemented. :P

Reply:
😆 In this case, I don't agree because it's something that only makes sense if we read it from the start, like a list... but that's only my 2 cents, it's up to @mmbotelho :)

Reply:
Well, okay then. I'm fine with it :)